src/ScribuntoLuaLibrary.php

```diff
@@ -56,7 +56,9 @@ public function getQueryResult( $queryString = null ) {
 		$api = new ApiMain( $params );
 		$api->execute();
 		$data = $api->getResult()->getResultData();
-
+		if(!empty($data) && !empty($data["query"]) && !empty($data["query"]["results"])) {
```

Comment:
I kept it as `!empty`, but you are correct that checking `$data["query"]["results"]` is sufficient. Change is at https://github.com/SemanticMediaWiki/SemanticScribunto/pull/6/commits/6b3dde989c73d1b03f3c320bc1407291b674a587